test/app.test.ts

```diff
@@ -13,4 +13,11 @@ describe('the application', (): void => {
 
     expect(response.status).toEqual(404);
   });
+
+  it('should support cross-origin requests', async (): Promise<void> => {
+    const response = await request(app.callback()).get('/').set('Origin', 'http://example.com');
+
+    expect(response.get('Access-Control-Allow-Origin')).toBe('*');
+    expect(response.get('Vary')).toContain('Origin');
```

Comment:
Need to check if this is an array, or still a comma-separated string.

Reply:
It varies only on the presence or absence of `Origin` right?

Reply:
No, value of...

Reply:
Relates to https://github.com/libero/article-store/pull/11#discussion_r345751382.